package.json

```diff
@@ -131,8 +131,11 @@
     "postcss-nesting": "^7.0.1",
     "preact": "^10.4.1",
     "prettier": "^2.0.4",
+    "prop-types": "^15.7.2",
     "pug": "^2.0.4",
     "puppeteer": "^3.0.0",
+    "react": "^16.13.1",
```

Comment:
Maybe you want to add these dependencies to playground? they should not be vite dependencies

Reply:
I think we should keep it here. Othewise the tests will be much slower, waiting for the `yarn install`: https://github.com/vitejs/vite/blob/963614bff7039207c0b2d9e8c2c5780b74ee176a/test/test.js#L49.

Reply:
Yes, i hope we will migrate to yarn workspaces some day